src/test/fixtures/profiles/gecko-profile.js

```diff
@@ -106,10 +106,22 @@ const thread = {
   markers: {
     schema: { name: 0, time: 1, data: 2 },
     data: [
-      [4, 0, { category: 'VsyncTimestamp', vsync: 0 }],
       [
-        5,
-        2,
+        // Ending a tracing but never starting it.
```

Comment:
This is an incomplete sentence and would be easier to understand as a full sentence.

Reply:
I used this, line 109:
```
      // Please keep the next marker at the start if you add more markers in
      // this structure.
      // We want to test tracing markers with the missing "start" marker. So
      // here is only the "end" marker without a matching "start" marker.
```